cypress/elements/catalog/assign-products.js

```diff
@@ -0,0 +1,6 @@
+export const ASSIGN_PRODUCTS_SELECTORS = {
+  searchInput: "[name='query']",
+  tableRow: "[data-test-id='assign-product-table-row']",
+  checkbox: "[type='checkbox']",
+  submitButton: "[type='submit']"
```

Comment:
Selectors of this kind will most likely be used in a lot of places, maybe worth extracting to something like `selectors.js` which could keep common ones.